Sources/TSCUtility/FSWatch.swift

```diff
@@ -428,55 +428,74 @@ public final class Inotify {
 // FIXME: <rdar://problem/45794219> Swift should provide shims for FD_ macros
 
 private func FD_ZERO(_ set: inout fd_set) {
+      #if os(Android)
+	set.fds_bits = (0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0)
+      #else
 	set.__fds_bits = (0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0)
+      #endif
 }
 
 private func FD_SET(_ fd: Int32, _ set: inout fd_set) {
     let intOffset = Int(fd / 16)
     let bitOffset = Int(fd % 16)
-    let mask = 1 << bitOffset
+    let mask: UInt = 1 << bitOffset
```

Comment:
I'd get errors that `mask` was set to `Int` otherwise, unsure why this varies for Android AArch64 but just specified the type.